packages/xo-server-perf-alert/src/index.js

```diff
@@ -478,13 +538,23 @@ ${monitorBodies.join('\n')}`
       snapshot: async () => {
         return Promise.all(
           map(
-            definition.smartMode
-              ? filter(
-                  this._xo.getObjects(),
-                  obj =>
-                    obj.type === objectType &&
-                    ((objectType !== 'VM' && objectType !== 'host') || obj.power_state === 'Running')
-                ).map(obj => obj.uuid)
+            definition.smartMode || definition.excludeUuids
+              ? filter(this._xo.getObjects(), obj => {
+                  if (obj.type !== objectType) {
+                    return false
+                  }
+
+                  let shouldKeepThisObj = true
+                  if (definition.smartMode) {
+                    shouldKeepThisObj = (objectType !== 'VM' && objectType !== 'host') || obj.power_state === 'Running'
+                  }
+
+                  if (definition.excludeUuids && shouldKeepThisObj) {
```

Comment:
Explicit test.

Reply:
`excludeUuids` can be `undefined` or a `boolean`. 
Do you want something like: `definition.excludeUuids === true`?